### ISSUE TITLE
Unable to get/set values for custom/system fields in Phone Call form

### ISSUE THREAD

**Issue by krishnacs1993** [Microsoft/EasyRepro#861]
Hi there,

I'm trying to get/set values of the fields (OOTB & Custom) present on the Phone Call activity form, but my tests always fail saying, field does not exist

Message: 
    Field with name directioncode does not exist.
  Stack Trace: 
    System.Exception: Field with name directioncode does not exist.

For "description" field, When I change my code to something below, it actually works 

xrmApp.Entity.SetValue(Elements.ElementId[Reference.Timeline.PhoneCallDescription], description);

**Comment by krishnacs1993** [comment]
@TYLEROL Can you please help me out here with the ways to set custom fields in Phone Call activity form.

**Comment by TYLEROL** [comment]
Hi @krishnacs1993 -

How are you creating the phone call specifically? If through the Timeline.AddPhoneCall() method, I have pushed a fix in #864 that should resolve some issues with that. 

```
// Sample from UCITestAccountAddPhoneCall()
xrmApp.Timeline.AddPhoneCall("Call Zoo primates section", "425-425-1235", "Specifically call the primate that can handle a phone !", "15 minutes");

xrmApp.QuickCreate.SetValue() // SetValue on your custom fields as needed here

xrmApp.Timeline.SaveAndClosePhoneCall();
```

If you are creating them via the Activities left-hand navigation men, then it looks like we'll need to add some new methods to extend existing SetValue() methods, but to set context to these custom dialog forms.

If this is regarding the custom dialog forms, please let me know.

Regards,
Tyler

**Comment by KumarSoniLove** [comment]
The xrmApp.Timeline.AddPhonecall works fine but if we want to use the same logic for creating a phone activity under Activities tab/entity, at the moment it does not work. Is there any way to work around or road map to fix that. Thanks
[ActivityScreenshot.docx](https://github.com/microsoft/EasyRepro/files/4694725/ActivityScreenshot.docx)
